searxng/searxng
# Various modification of the documentation and the theme sphinx-theme

## What does this PR do?

Various modification of the documentation and the theme sphinx-theme.

For any details, read along the commit messages.

## How to test this PR locally?

    make docs.clean docs.live

You should not see any error massage in the build log of the documentation.

Have a look at the produce HTML .. the logo has been changed ...

![grafik](https://user-images.githubusercontent.com/554536/142467144-2824b37d-d12c-4f40-8da5-2fea4bb1576b.png)

and there is a "source" link in the sidebar.

![grafik](https://user-images.githubusercontent.com/554536/142467189-5a9f3242-2096-4c5d-b81a-32aa50b38ee2.png)
